autoload/lsp/completion.vim

```diff
@@ -590,12 +590,17 @@ def LspCompleteDone(bnr: number)
     # additional text edits.  So try to resolve the completion item now to get
     # the text edits.
     completionData = lspserver.resolveCompletion(completionData, true)
-    if completionData->get('additionalTextEdits', {})->empty()
-      return
+    if !completionData->get('additionalTextEdits', {})->empty()
+      textedit.ApplyTextEdits(bnr, completionData.additionalTextEdits)
     endif
   endif
 
-  textedit.ApplyTextEdits(bnr, completionData.additionalTextEdits)
+  if completionData->has_key('command')
```

Comment:
what if it had both (just in case) ?

Reply:
and is it possible or what if it was delayed (or not) to get  it like additionaltextedits ?

Reply:
> what if it had both (just in case) ?

Then both should be used. The command can be something completely unrelated to the additionaltextedit and the LSP spec mandates that we apply the command.

> and is it possible or what if it was delayed (or not) to get it like additionaltextedits ?

Do you know a specific language server that does this or is this just speculation? Unlike the textedit, the command name is not expensive to compute, so I don't know any reason why a server would delay it.

Reply:
ok, if you were sure about it.